mrrc/__init__.py

```diff
@@ -24,6 +24,7 @@
 cli.add_command(init)
 cli.add_command(upload)
 cli.add_command(delete)
+cli.add_command(index)
```

Comment:
We will not have a separate index command here. The index process will be embedded into the upload & delete process after the files uploaded to/deleted from s3.